File: 백준/Bronze/10886. 0 ＝ not cute ／ 1 ＝ cute/0 ＝ not cute ／ 1 ＝ cute.py
k = int(input())

v1 = 0
v0 = 0
for i in range(k):
    n = int(input())
    if n == 1:
        v1 += 1
    else:
        v0 += 1

if v1 > v0:
    print("Junhee is cute!")
else:
    print("Junhee is not cute!");    

File: 백준/Bronze/10886. 0 ＝ not cute ／ 1 ＝ cute/0 ＝ not cute ／ 1 ＝ cute.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <math.h>
#include <ctype.h>
#include <limits.h>

#define TRUE 1 
#define FALSE 0

int main() {
    int N;
    scanf("%d", &N);

    // no, yes
    int arr[2] = {0, 0};


    for(int i = 0; i < N; i++){
        int num;
        scanf("%d", &num);

        if(num == 0){
            arr[0]++;
        }else{
            arr[1]++;
        }
    }

    if(arr[0] > arr[1]){
        printf("Junhee is not cute!\n");
    }else{
        printf("Junhee is cute!\n");
    }

    return 0;
}